cardano-cli/golden/files/golden/help/governance_action_create-new-committee.cli

```diff
@@ -0,0 +1,15 @@
+Usage: cardano-cli governance action create-new-committee --epoch EPOCH
+            --drep-cold-verification-key-file FILE
+            [--cost-model-file FILE]
+            --out-file FILE
+
+  Create a new-committee action
+
+Available options:
+  --epoch EPOCH            The epoch number in which the update proposal is
+                           valid.
+  --drep-cold-verification-key-file FILE
```

Comment:
any ada holder can create governance actions. We need to change this to use stake.vkey